src/staking-v3/components/DiscoverV3.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div class="wrapper--discover">
+    <!-- Todo: Delete -->
+    <!-- <staking-home /> -->
+    <feature-dapp />
+    <!-- <staking /> -->
+    <leaderboard />
+    <leaderboard-vote />
+    <!-- <ad /> -->
```

Comment:
@bobo-k2 Can I remove this components?

Reply:
`Staking` component is used on assets page and I would like to use it here, so please leave the component. I don't know what `ad` component is about. Please check and remove if needed.

Reply:
Ok, fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/1023/commits/13f5be14427b6e71996af675199f84ed683d3a08) commit.